docs/eventing/samples/github-source/README.md

```diff
@@ -37,7 +37,7 @@ spec:
       revisionTemplate:
         spec:
           container:
-            image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/message_dumper
+            image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display
 ```
 
 Enter the following command to create the service from `service.yaml`:
```

Comment:
Similar comment on line 150. 

I would replace the entire `Create Events` command with:

```shell
kubectl logs -l serving.knative.dev/service=event-display -c user-container --since=10m
```